components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/java/org/wso2/carbon/identity/configuration/mgt/endpoint/util/ConfigurationEndpointUtils.java

```diff
@@ -307,10 +309,24 @@ public static InternalServerErrorException buildInternalServerErrorException(Str
         return new InternalServerErrorException(errorDTO);
     }
 
-    public static <T> Condition getSearchCondition(SearchContext searchContext, Class<T> reference) {
+    public static <T> Condition getSearchCondition(SearchContext searchContext, Class<T> reference, Log log)
```

Comment:
Any reason for passing the log here ?

Reply:
Fixed in e4e5d28c2d9caf6269b0ba6086fdb4203783ebbb